### PR TITLE
An initial debian package version

### PR DESCRIPTION
**Pull request by jsreynaud**
Adding files to build .deb packages for this project. For the moment tested only on Ubuntu focal (20.04)

### CODE REVIEW

**Comment by halbux** [comment]
Hi Jean-Samuel,

Thanks! For your packaging help, really appreciated!

Will I be able to check everything is working before the package is submitted to debian?

**Comment by jsreynaud** [comment]
Hi Alexandre,

For the moment I only propose you to add packaging files for debian. I don't plan to manage submission on Debian archive. I propose that because there is some works on KiCad that use your lib and I manage PPA (Ubuntu daily builds) for KiCad. If KiCad main branch use your lib I will have to find a way to add your lib during the build process...

Anyway I'm ok to propose your lib to debian archive (and maintain if accepted and if nobody want to maintain).

So to summarize: This PR is only an initial version and you have time to check everything before any submission ...
